src/diffusers/loaders.py

```diff
@@ -957,6 +978,7 @@ def lora_state_dict(
         revision = kwargs.pop("revision", None)
         subfolder = kwargs.pop("subfolder", None)
         weight_name = kwargs.pop("weight_name", None)
+        unet_config = kwargs.pop("unet_config", None)
```

Comment:
@patrickvonplaten how is this supposed to be passed?

```python
pipeline = StableDiffusionXLPipeline(...)
pipeline.load_lora_weights(".", weight_name=..., unet_config=pipeline.unet.config)
```

Is this what you envisioned?

Reply:
Nevermind. Saw an example.